CMakeLists.txt

```
@@ -2688,7 +2688,7 @@ set(CPACK_PACKAGE_EXECUTABLES "mixxx;Mixxx")
 set(CPACK_PACKAGE_ICON "${CMAKE_SOURCE_DIR}/res/images/mixxx_install_logo.bmp")
 set(CPACK_PACKAGE_HOMEPAGE_URL "https://www.mixxx.org/")
 if(NOT GIT_DESCRIBE)
-  set(CPACK_PACKAGE_FILE_NAME "mixxx-${CMAKE_PROJECT_VERSION}-unknown")
+  set(CPACK_PACKAGE_FILE_NAME "mixxx-${MIXXX_VERSION}")
```

```suggestion
  set(CPACK_PACKAGE_FILE_NAME "mixxx-${MIXXX_VERSION}-unknown")
```
To stress that we don't know the exact version? Because otherwise it's hard to distinguish from Git describe when on a tag.

---

But i don't care, we can also merge as-is.

---

It is a rare case anyway.
Let's make it ugly to encourage using git